src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -318,13 +323,20 @@
     default-so-linger-in-milliseconds))
 
 (schema/defn ^:always-validate
+  common-connector-config :- WebserverConnectorCommon
+  [config :- WebserverRawConfig]
+  {:request-header-max-size   (or (:request-header-max-size config)
+                                  default-request-header-size)
+   :so-linger-milliseconds    (so-linger-in-milliseconds config)
+   :idle-timeout-milliseconds (:idle-timeout-milliseconds config)})
```

Comment:
We should have a more broad discussion about this with the team; let's figure out how to get that scheduled.

I'm firmly in the "no" camp here; I like the idea of making things configurable but I'm pretty much terrified of making assumptions that we will have more insight into what the sane default values should be than the upstream authors would.

Think about this in light of the tuning work we're about to do in Puppet Server.  If someone had written their own wrapper around our installer that resulted in a hard-coded override of our internal defaults, to set them to what we currently set them to, and then we roll out Puppet Server 1.1 with a bunch of improvements to those defaults based on our own domain expertise, they wouldn't get those changes.  And that'd be really bad.

I feel like that's exactly what we're doing with a bunch of our Jetty settings, and it makes me nervous.

The main argument that would make me consider hard-coding our own default in for this new setting is that we're already doing it for a lot of the other settings, and as I was writing this code, it felt inconsistent because I wasn't doing it that way.  So I might be willing to change it just for the sake of consistency, but overall the pattern still feels overly risky to me.